openfe_gromacs/protocols/gromacs_md/md_methods.py

```diff
@@ -245,14 +245,18 @@ def get_filenames_em(self) -> dict[str, list[pathlib.Path]]:
             "cpt_em",
         ]
         dict_em = {}
+
         for file in file_keys:
+            try:
+                file_path = [
+                    pus[0].outputs[file]
+                    for pus in self.data.values()
+                    if "GromacsMDRunUnit" in pus[0].source_key
+                ]
+                dict_em[file] = file_path
+            except KeyError:
+                dict_em[file] = None
 
-            file_path = [
-                pus[0].outputs[file]
-                for pus in self.data.values()
-                if "GromacsMDRunUnit" in pus[0].source_key
-            ]
-            dict_em[file] = file_path
         return dict_em
 
     def get_gro_em_filename(self) -> list[pathlib.Path]:
```

Comment:
These names are a bit confusing, they should be plural since you return a list of paths not a single path,

Reply:
Done